### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by cnevakivicallanan**
I am generally a little bit confused about how this all works but am confident that it'll all get easier. Huge shoutout to the TAs, you guys are so helpful.

P.S. My secondary bullet points were running great in R Studio, but don't appear to be showing up in the preview on GitHub... please let me know if you happen to have any insight into this. Thanks!

Cleo Nevakivi-Callanan

### CODE REVIEW

**Comment by cscrivner** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |          |       x       |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:
Great job! Nice bunny pics. As for the list, it might work if you add 2 spaces after the + sign. Nice job doing more than the bare minimum for the assignment. 

Pretty cool that you were born in Helsinki - have you been back?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+### Interests
+In my free time, I enjoy reading (non-academic books), visiting museums, playing violin, drawing, and doing yoga.
+
+Also... **I LOVE BUNNIES!**
+
+Here is a bunny, that I love:
+
+![Ready for Coachella](https://i2.wp.com/mainetoday.com/wp-content/uploads/2018/10/554956-Bunny-Cuddling.jpg?fit=800%2C533)
+
+Also this very handsome boi:
```
Comment (kelseypagorek) [inline README.md:17]: THANK YOU FOR SHARING THIS HANDSOME BOI.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+
+![Ready for Spring](https://mymodernmet.com/wp/wp-content/uploads/archive/TS39l1veeVBLqUnbvRYj_mumitan2.jpg)
+
+*TL;DR I love bunnies*
+
+[Here are some fun facts about bunnies!](http://mentalfloss.com/article/557234/facts-about-rabbits)
+
+### Computer Science and all that
```
Comment (kelseypagorek) [inline README.md:25]: Nice list! I think you did a good job explaining where you struggled and how you powered through. You mentioned not knowing much about computers, but it already seems like you've already learned a lot!

**Review by kelseypagorek — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |


Great job! My favorite part was the handsome boi picture. Thank you for explaining your roadblocks so clearly - I definitely shared some of your troubles, so it is nice to know I was not alone!

**Comment by dmukkamala** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |        X   |              |            |
| **Presentation**            |       X    |              |            |
| **Achievement, creativity** |        X   |              |            |
| **Ease of access**          |       X    |              |            |


Excellent job on Homework 1. You went above and beyond in adding quite a bit of depth to your biography; it demonstrates a strong competence. Thank you, also, for the fun facts about bunnies; the list was a highlight of my day. I had similar struggles with the link from RStudio to Git and resolved my issue after consulting the notes. You did a great job diagramming your process step-by-step.

**Pull request closed without merging**